packages/desktop-client/src/components/modals/FixEncryptionKey.js

```diff
@@ -127,7 +127,8 @@ export default function FixEncryptionKey({
               <Button
                 style={{ marginRight: 10 }}
                 onClick={() => modalProps.onBack()}
-                type="button"
+                type="normal"
+                isSubmit={false}
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏setting `isSubmit` is unnecessary here

Reply:
Ah yes you're right

Reply:
@MatissJanis updated!